test/image/mocks/zz-subtitle.json

```diff
@@ -0,0 +1,27 @@
+{
+    "data": [
+        {
+            "x": [1, 2, 3, 4],
+            "y": [10, 12, 11, 13],
+            "type": "scatter"
+        }
+    ],
+    "layout": {
+        "title": {
+            "text": "A simple plot",
+            "subtitle": {
+                "text": "With a subtitle"
```

Comment:
Nice. If I use a mathjax e.g. `$E^2=m^2c^4+p^2c^2$` in the subtitle text it works.
So you may want to add a new mock for that case.
But if I set a mathjax to the title the position of subtitle is not correct. That's something to keep in mind to revisit later.

Reply:
Good catch, I'll look into this.

Reply:
Great that now one could use mathjax in both subtitle and title. :medal_sports: 
Let's use mathjax in subtitle and/or title of these mocks:
```js
`table_plain_birds`, 
`table_wrapped_birds`, 
`ternary-mathjax`, 
`table_latex_multitrace_scatter`.
```
Please note that we using mathjax v3 these mocks render slightly differently and extra baseline beginning with `mathjax3___` are generated on CircleCI which you could collect from artifatcs of `test-baselines-mathjax3` artifacts and replace them in the `baselines` folder.
Thank you!

Reply:
Please consider adding mathjax subtitles to those mocks. 
Thank you!